perun-base/src/main/java/cz/metacentrum/perun/core/api/MembersPageQuery.java

```diff
@@ -96,6 +109,22 @@ public void setStatuses(List<Status> statuses) {
 		this.statuses = statuses;
 	}
 
+	public Group getGroup() {
+		return group;
+	}
```

Comment:
I didn't notice this earlier we we definitely want to pass here only groupId, not the whole group object. Please, change this to integer and also, don't forget to update the openapi.

Reply:
Fixed.